src/openmc_driver.cpp

```diff
@@ -21,6 +20,31 @@ OpenmcDriver::OpenmcDriver(MPI_Comm comm)
     err_chk(openmc_init(0, nullptr, &comm));
   }
   MPI_Barrier(MPI_COMM_WORLD);
+
+  // determine number of fissionable cells in model to aid in catching
+  // improperly mapped problems
+  n_fissionable_cells_ = 0;
+  for (int i = 0; i < cells_size(); ++i) {
```

Comment:
```suggestion
  for (gsl::index i = 0; i < openmc::model::cells.size(); ++i) {
```
This will also require
```C++
#include "openmc/cell.h"
```

Reply:
Ah, not sure why I didn't think of that...